etc/hyperchains/docker-compose-hyperchain-template

```diff
@@ -0,0 +1,158 @@
+# System requirements:
+# ~16+ CPU cores
+# ~384+ GB RAM
+# This assumes the following:
+# - This Docker Compose is running from https://github.com/matter-labs/zksync-era/tree/boojum-integration
+# - Hyperchain Wizard was already ran and common.env file is generated in this dir
+# - DB is available/ready/initialized and common.env is pointing to it
+# - Contracts are already built and setup key is downloaded (most of the things should be done in Hyperchain Wizard)
+# reference: https://github.com/matter-labs/zksync-era/blob/main/.github/workflows/build-core-template.yml#L67-L72
+# - Data keys are already downloaded
+# from: https://console.cloud.google.com/storage/browser/matterlabs-zksync-v2-infra-blob-store/prover_setup_data/2d33a27?pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&orgonly=true&project=matterlabs-infra&supportedpurview=organizationId&prefix=&forceOnObjectsSortingFiltering=false
```

Comment:
Is this URL correct? I tried using it in incognito mode and it asks me to log in. Shouldn't it be a public one?

Reply:
@githubdoramon i'd rather remove everything from here

Reply:
@hatemosphere ?

Reply:
ah, damn, didn't see your comment before pushing